<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Light_tool</class>
 <widget class="QWidget" name="Light_tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>781</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="light_1_label">
      <property name="text">
       <string>Light 1</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="light_1_rot_lable">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>5</width>
          <height>3</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="light_1_rot_x_line">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="light_1_rot_z_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_1_rot_y_line"/>
      </item>
      <item>
       <widget class="QLabel" name="light_1_dist_label">
        <property name="text">
         <string>Distance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="light_1_dist_x_line">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="light_1_dist_z_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_1_dist_y_line"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="light_2_label">
      <property name="text">
       <string>Light 2</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="light_2_rot_label">
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="light_2_rot_x_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_2_rot_z_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_2_rot_y_line"/>
      </item>
      <item>
       <widget class="QLabel" name="light_2_dist_label">
        <property name="text">
         <string>Distance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="light_2_dist_x_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_2_dist_z_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_2_dist_y_line"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="light_3_label">
      <property name="text">
       <string>Light 3</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="light_3_rot_label">
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="light_3_rot_x_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_3_rot_z_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_3_rot_y_line"/>
      </item>
      <item>
       <widget class="QLabel" name="light_3_dist_label">
        <property name="text">
         <string>Distance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="light_3_dist_x_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_3_dist_z_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="light_3_dist_y_line"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
      <item>
       <widget class="QPushButton" name="three_lights_button">
        <property name="text">
         <string>Confirm Lights</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
